code/src/App.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react'
+import { BrowserRouter, Routes, Route } from "react-router-dom"
+
+import MovieList from 'Components/MovieList'
+import MovieDetails from 'Components/MovieDetails'
+import Error from 'Components/Error'
+
 
 export const App = () => {
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <div>
+        <Routes>
+            <Route path='/' element={<MovieList />} />
+            <Route path='/moviedetails/:movieId' element={<MovieDetails />} />
+            <Route path='*' element={<Error />} />
```

Comment:
This is how I and Lovisa did the handling of errors, check it out if you are interested :) 
   ```
     <Route path="/404" element={<NotFond />} />
        <Route path="*" element={<Navigate to="/404" replace />} />
```

Reply:
Hi! Thanks for your comment. With the path of '*' the react router will render all other paths to the error page. It works the same as your solution.

> This is how I and Lovisa did the handling of errors, check it out if you are interested :)
> 
> ```
>   <Route path="/404" element={<NotFond />} />
>      <Route path="*" element={<Navigate to="/404" replace />} />
> ```